From aff15d704012b630b41b4bf6e871a6cf16ab18ec Mon Sep 17 00:00:00 2001
From: Alexander Mikhalitsyn <alexander.mikhalitsyn@virtuozzo.com>
Date: Tue, 21 Dec 2021 23:42:26 +0300
Subject: [PATCH 238/249] Revert "ci: disable glibc rseq support"

Let's see how rseq() C/R feature works

This reverts commit d99def7dcfa938918368c91021f72a77f738bc61.

Signed-off-by: Alexander Mikhalitsyn <alexander.mikhalitsyn@virtuozzo.com>
---
 scripts/build/Dockerfile.fedora.tmpl | 3 ---
 scripts/ci/run-ci-tests.sh           | 7 +------
 2 files changed, 1 insertion(+), 9 deletions(-)

diff --git a/scripts/build/Dockerfile.fedora.tmpl b/scripts/build/Dockerfile.fedora.tmpl
index fd4ba4aef..9d3bb0f87 100644
--- a/scripts/build/Dockerfile.fedora.tmpl
+++ b/scripts/build/Dockerfile.fedora.tmpl
@@ -1,8 +1,5 @@
 ARG CC=gcc
 
-# FIXME: Temporary solution for https://github.com/checkpoint-restore/criu/issues/1696
-ENV GLIBC_TUNABLES=glibc.pthread.rseq=0
-
 COPY scripts/ci/prepare-for-fedora-rawhide.sh /bin/prepare-for-fedora-rawhide.sh
 RUN /bin/prepare-for-fedora-rawhide.sh
 
diff --git a/scripts/ci/run-ci-tests.sh b/scripts/ci/run-ci-tests.sh
index 14e6b1224..f89783630 100755
--- a/scripts/ci/run-ci-tests.sh
+++ b/scripts/ci/run-ci-tests.sh
@@ -255,12 +255,7 @@ if [ -n "$TRAVIS" ] || [ -n "$CIRCLECI" ]; then
        # Error (criu/tty.c:1014): tty: Don't have tty to inherit session from, aborting
        make -C test/others/shell-job/ run
 fi
-
-# FIXME: rpc tests fail even with set glibc tunable
-# https://github.com/checkpoint-restore/criu/issues/1696
-if [ "$GLIBC_TUNABLES" != "glibc.pthread.rseq=0" ]; then
-	make -C test/others/rpc/ run
-fi
+make -C test/others/rpc/ run
 
 ./test/zdtm.py run -t zdtm/static/env00 --sibling
 
-- 
2.35.1

